js/index.js

```diff
@@ -14,18 +36,42 @@ function calculateAll() {
   // end of test
 
   // ITERATION 2
-  //... your code goes here
+  let totalSum = updateSubtotal();
 
   // ITERATION 3
-  //... your code goes here
+  total.innerHTML = totalSum;
```

Comment:
Aqui a função não está imprimindo o valor total na tela, conforme esperado. 
Você pode pegar todos os produtos pela classe, fazer um for e ir somando os valores. Em seguida passar o resultado para o span do total no HTML, usando innerHTML.